lib/App/DuckPAN/Perl.pm

```diff
@@ -109,54 +111,73 @@ sub duckpan_install {
 		$sp =~ s/\:\:/_/g;
 
 		# special case: check for a pinned verison number
-		my $pin_version            = $ENV{$sp};
-		my $localver               = $self->get_local_version($package);
-		my $duckpan_module_version = version->parse($module->version);
-		my $duckpan_module_url     = $self->app->duckpan . 'authors/id/' . $module->distribution->pathname;
+		my $pinned_version      = $ENV{$sp};
+		my $installed_version   = $self->get_local_version($package);
+		my $latest_version      = version->parse($module->version);
+		my $duckpan_module_url  = $self->app->duckpan . 'authors/id/' . $module->distribution->pathname;
 
-		$localver ||= 1e-6 if ($pin_version); # a silly, but true, value if missing and we need to compare with pinned.
```

Comment:
I guess the name of the function should really be "reset" not so much "reinstall". It's meant to get a DDG dev back to where prod is.

Right this moment, I'm in need of this function. I've been doing both Spice and Goodie builds (and testing this PR) and so my Spice, Goodie and DuckPAN repos are out of sync with prod. Now that I want to test a prod release of the IA's I need to "reinstall" the currently released repos and so I would typically run "duckpan reinstall" to get all my repos back to where prod is at.

Does that clarify things?